--- web/core/lib/Drupal/Core/Recipe/ConfigConfigurator.php	2024-09-12 11:38:10.000000000 +0200
+++ ConfigConfigurator.php	2024-12-19 10:38:27.798682628 +0100
@@ -43,9 +43,9 @@
         //   fixed in core, use that API instead to sort the config data.
         self::recursiveSortByKey($active_data);
         self::recursiveSortByKey($recipe_data);
-        if ($active_data !== $recipe_data) {
-          throw new RecipePreExistingConfigException($config_name, sprintf("The configuration '%s' exists already and does not match the recipe's configuration", $config_name));
-        }
+        #if ($active_data !== $recipe_data) {
+        #  throw new RecipePreExistingConfigException($config_name, sprintf("The configuration '%s' exists already and does not match the recipe's configuration", $config_name));
+        #}
       }
     }
   }
